<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProviderPage</class>
 <widget class="QWidget" name="ProviderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select your provider from the list below or click advanced if your provider is not listed</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KFilterProxySearchLine" name="searchLine"/>
   </item>
   <item>
    <widget class="QTreeView" name="listView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="advancedButton">
       <property name="text">
        <string>Advanced</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
